package.json

```diff
@@ -37,41 +38,43 @@
   "keywords": [],
   "author": "Tobias Golbs <golbs@e-spirit.com>",
   "devDependencies": {
-    "@commitlint/cli": "^9.1.2",
-    "@commitlint/config-conventional": "^9.1.2",
-    "@nuxt/types": "^2.14.1",
+    "@commitlint/cli": "^12.1.4",
+    "@commitlint/config-conventional": "^12.1.4",
+    "@nuxt/types": "^2.15.7",
     "@semantic-release/changelog": "^5.0.1",
     "@semantic-release/commit-analyzer": "^8.0.1",
     "@semantic-release/git": "^9.0.0",
-    "@semantic-release/github": "^7.0.7",
-    "@semantic-release/npm": "^7.0.5",
-    "@semantic-release/release-notes-generator": "^9.0.1",
-    "@types/jest": "^26.0.9",
+    "@semantic-release/github": "^7.2.3",
+    "@semantic-release/npm": "^7.1.3",
+    "@semantic-release/release-notes-generator": "^9.0.3",
+    "@types/jest": "^26.0.23",
     "@types/lodash.merge": "^4.6.6",
-    "@types/node": "^14.0.27",
-    "@typescript-eslint/eslint-plugin": "^3.9.1",
-    "@typescript-eslint/parser": "^3.9.1",
-    "commitizen": "^4.1.2",
+    "@types/node": "^16.0.1",
```

Comment:
Deployment target of most fsxa-pwa is node 14. Did you test, does the node module still work for node 14? Is there an automated test for it?